### PR TITLE
Doc - adds info about user to be used to start the starter

### PR DESCRIPTION
**Pull request by sleto-it**
Now ready for review / merging

### CODE REVIEW

**Comment by Simran-B** [comment]
Changed formatting and style a bit, improved cross-reference labels

File: docs/Manual/Deployment/ActiveFailover/UsingTheStarter.md [hunk 1]
```diff
@@ -39,10 +39,10 @@ arangodb --starter.mode=activefailover --server.storage-engine=rocksdb --starter
 
 Run the above command on machine A, B & C.
 
-The _Starter_ will decide on which 2 machines to run a single server instance.
-To override this decision (only valid while bootstrapping), add a
-`--cluster.start-single=false` to the machine where the single server
-instance should _not_ be started.
+An Agent and a Single Instance will be started on each machine. One of the Single Instance will
```
Comment (Simran-B) [inline docs/Manual/Deployment/ActiveFailover/UsingTheStarter.md]: One of the Single Instance<ins>s</ins> (plural) I think?

File: docs/Manual/Deployment/ActiveFailover/UsingTheStarter.md [hunk 2]
```diff
@@ -39,10 +39,10 @@ arangodb --starter.mode=activefailover --server.storage-engine=rocksdb --starter
 
 Run the above command on machine A, B & C.
 
-The _Starter_ will decide on which 2 machines to run a single server instance.
-To override this decision (only valid while bootstrapping), add a
-`--cluster.start-single=false` to the machine where the single server
-instance should _not_ be started.
+An Agent and a Single Instance will be started on each machine. One of the Single Instance will 
+have the _leader_ role, the others two the _follower_ role.  
+If you want to have only one follower, add (only valid while bootstrapping),
```
Comment (Simran-B) [inline docs/Manual/Deployment/ActiveFailover/UsingTheStarter.md]: only **possible** while bootstrapping ? Maybe move this remark to the end of the sentence?

**Comment by sleto-it** [comment]
Adding Kaveh for a quick review from the Cluster Team.

Kaveh, in the PR there are a few different changes (you can basically ignore them I believe), but the main point of this PR is the following:

We are suggesting to not run the _arangodb_ using root, but using a limited privileged user, so the command we are suggesting is this one (assuming the user is _arangodb_:

```
sudo su arangodb -s /bin/bash -c "arangodb --all.server.uid=$(id -u arangodb) ... AnyStarterOptionHere"
```

When possible, please let us know if you believe this approach is okay, or if you have any feedback. As soon as this is merged, this will be incorporated into the official documentation and also Support will start suggesting this approach

If other approaches are needed - please let us know

Thanks!!

File: docs/Manual/Deployment/ActiveFailover/UsingTheStarter.md [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
 ==========================
 
-This section describes how to start an Active Failover setup the tool [_Starter_](../../Programs/Starter/README.md)
-(the _arangodb_ binary program).
+This section describes how to start an _Active Failover_ setup using the tool
+[_Starter_](../../Programs/Starter/README.md) (the _arangodb_ executable).
+
+{% hint 'warning' %}
+It is **strongly suggested** to **not** run the _ArangoDB Starter_ process
+(_arangodb_) from a system _user_ with full privileges, e.g. _root_ under Linux.
```
Comment (Simran-B) [inline docs/Manual/Deployment/ActiveFailover/UsingTheStarter.md]: I don't see why "user" should be italic
```suggestion
(_arangodb_) as system user with full privileges, e.g. _root_ under Linux.
```

File: docs/Manual/Deployment/ActiveFailover/UsingTheStarter.md [hunk 4]
```diff
@@ -1,8 +1,17 @@
...
+
+We suggest to create a dedicated _user_ with limited privileges for this purpose.
```
Comment (Simran-B) [inline docs/Manual/Deployment/ActiveFailover/UsingTheStarter.md]:
```suggestion
We suggest to create a dedicated system user with limited privileges for this purpose.
```

File: docs/Manual/Deployment/ActiveFailover/UsingTheStarter.md [hunk 5]
```diff
@@ -1,8 +1,17 @@
...
+Please refer to the [Linux user used to run _arangodb_](../ArangoDBStarter/README.md#linux-user-used-to-run-arangodb)
+section of the Starter Deployment page for further details.
```
Comment (Simran-B) [inline docs/Manual/Deployment/ActiveFailover/UsingTheStarter.md]: Can we add some info for macOS and Windows as well?

File: docs/Manual/Deployment/ArangoDBStarter/README.md [hunk 6]
```diff
@@ -15,3 +15,18 @@ Deploying using the ArangoDB Starter
...
 - [_Starter_ using processes](../Cluster/UsingTheStarter.md)
 - [_Starter_ using Docker containers](../Cluster/UsingTheStarter.md#using-the-arangodb-starter-in-docker)
+
+Best Practices
+--------------
+
+### Linux user used to run _arangodb_
+
+It is strongly suggested to not run the _ArangoDB Starter_ process (_arangodb_)
+as _root_. We suggest creating a dedicated linux _user_ for this purpose.
```
Comment (Simran-B) [inline docs/Manual/Deployment/ArangoDBStarter/README.md]:
```suggestion
as _root_. We suggest creating a dedicated Linux user for this purpose.
```

File: docs/Manual/Deployment/ActiveFailover/UsingTheStarter.md (hunk 5 above)
Comment (sleto-it) [reply docs/Manual/Deployment/ActiveFailover/UsingTheStarter.md]: Simran - can you?

Comment (Simran-B) [reply docs/Manual/Deployment/ActiveFailover/UsingTheStarter.md]: On Windows in a Command Prompt or PowerShell you can do `runas /user:arangodb "arangodb args"` but I couldn't figure out how to escape certain characters. Is our general recommendation to use a restricted system user account on all platforms? I know that MySQL creates a separate (hidden?) account, which has different file & folder permissions for instance. I don't know what type of account it actually is however.

Comment (sleto-it) [reply docs/Manual/Deployment/ActiveFailover/UsingTheStarter.md]: please can you open a ticket for this? we incorporate it in a later PR - thanks

File: docs/Manual/Deployment/ArangoDBStarter/README.md [hunk 7]
```diff
@@ -15,3 +15,18 @@ Deploying using the ArangoDB Starter
...
+
+It is strongly suggested to not run the _ArangoDB Starter_ process (_arangodb_)
+as _root_. We suggest creating a dedicated Linux user for this purpose.
+
+Assuming the user you want to use is called _arangodb_, to run the _Starter_,
+on Linux systems, you might use a command similar to the following:
+
+```
+sudo su arangodb -s /bin/bash -c "arangodb --all.server.uid=$(id -u arangodb) ... AnyStarterOptionHere"
+```
```
Comment (neunhoef) [inline docs/Manual/Deployment/ArangoDBStarter/README.md]: I suggest to add a hint that for production use one should use something like a `systemd` service to run the starter. There one can configure the user under which it is run easily and this makes sure that things are restarted automatically, for example after a reboot. Details can be found here: https://www.arangodb.com/2019/01/run-multiple-versions-arangodb/

File: docs/Manual/Programs/Starter/README.md [hunk 8]
```diff
@@ -1,7 +1,20 @@
...
 
-This chapter documents the _ArangoDB Starter_.
+The _ArangoDB Starter_ is a tool that can help you deploy ArangoDB in an easy way
+(either in _Single Instance_, _Active Failover_ or _Cluster_ mode).
 
-The _ArangoDB Starter_ is a tool that can help you deploy ArangoDB in an easy way (either in single-instance, active/passive or Cluster mode).
+The binary is called `arangodb`, not to be confused with the server binary `arangod`.
 
-For a Tutorial, please refer to [this](../../Tutorials/Starter/README.md) section.
+Also see:
```
Comment (neunhoef) [inline docs/Manual/Programs/Starter/README.md]: This is a bit strange use of the English language. I think "See also:" is more common,

**Comment by nikita-vanyasin** [comment]
@Simran-B can you please confirm if this PR is still needed? I think all actual documentation for starter was moved to main docs web-site long time ago.

File: docs/Manual/Programs/Starter/README.md (hunk 8 above)
Comment (jwierzbo) [inline docs/Manual/Programs/Starter/README.md:8]:
```suggestion
See also:
```

**Pull request closed without merging**